packages/cli/src/cli/index.ts

```diff
@@ -62,6 +62,7 @@ $$$$$$\\   $$$$$$$\\  $$\\  $$$$$$\\   $$$$$$$ |$$\\  $$\\  $$\\  $$$$$$\\  $$$$
       `Explicitly tell the CLI to bootstrap the app using pnpm`,
     )
     .option("--framework [name]", `The preferred framework.`)
+    .option("--solana", `Initialize as a Solana project.`)
```

Comment:
this should likely only work in combination with `--app`

`npx thirdweb create --app --solana` makes sense 

`--solana` top level (same as `--app` seems wrong in terms of hierarchy